docs/develop/nova/additional-guides/foundry_guide.md

```diff
@@ -119,7 +119,7 @@ keywords:
     Environment variables can include database connection details, API keys, external resource URIs, or other configuration variables that might change depending on the environment in which the application is running. In our case, we would use it to point to our Core-EVM RPC url by setting
 
     ```bash
-    RPC_URL=https://nova-0.gemini-3h.subspace.network/ws
+    RPC_URL=wss://rpc.taurus.subspace.foundation/ws
```

Comment:
This is not up yet 😁

Reply:
Thank @EmilFattakhov. I just created a new PR: https://github.com/autonomys/subspace-docs/pull/718 to revert the line.